x-pack/plugin/monitoring/src/main/java/org/elasticsearch/xpack/monitoring/exporter/http/HttpExportBulk.java

```diff
@@ -162,7 +161,7 @@ protected void doClose(ActionListener<Void> listener) {
             // Adds final bulk separator
             out.write(xContent.streamSeparator());
 
-            logger.trace("added index request [index={}, type={}, id={}]", index, doc.getType(), id);
+            logger.trace("http exporter [{}] - added index request [index={}, id={}]", name, index, id);
```

Comment:
The type being logged here wasn't `_type` but the user-defined `type` field inside `.monitoring-*` docs. I removed it to prevent confusion. Happy to add it back if reviewers think its worth keeping.

Reply:
The logging is a lot less useful without the type because you can't make heads or tails of anything except `shard` documents because they're the only ones that specify an `id`.

Reply:
Thanks, I'll add the type back but I'd prefer to call it something like `monitoring data type` so it's slightly clearer that it's not related to `_type`. Especially since the other two strings being logged here are `_index` and `_id`.